tests/test_usi.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import Mock
+from unittest.mock import patch
+import numpy as np
+from matchms import Spectrum
+from matchms.importing import load_from_usi
+
+
+@patch('requests.get')
+def normal_test(mock_get):
```

Comment:
Test is ignored as pytest needs methods to start with `test_`. This causes low coverage and SonarCloud failure.
```suggestion
def test_normal(mock_get):
```